website/templates/index.mako

```diff
@@ -42,6 +42,57 @@
           <div class="circle" id="circle-7"><span></span></div>
           <div class="circle" id="circle-8"><span></span></div>
         </div>
+        <div id="signUp" class="anchor"></div>
```

Comment:
Is this file supposed to be in this PR? It doesn't seem to be aligned with its overall purpose.